ports/osgearth/vcpkg.json

```
@@ -1,9 +1,10 @@
 {
   "name": "osgearth",
-  "version": "3.2",
-  "port-version": 5,
+  "version": "3.3",
+  "port-version": 6,
```

```suggestion
  "version": "3.3",
```
Thanks for your PR! Can you please correct the Version field in the CONTROL file? You can get more information from [maintainer-guide.md](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md).